### DISCUSSION TITLE
Is there a helper to get related browsers from the model

### DISCUSSION THREAD

**Discussion by michaeljcoyne** [area17/twill#1920] [General]
Hi is there a helper to get the related browser from the model to save looking it up (n+1) per item?

So insted of 

```$speaker->getRelated('featuredEvent');```

Something like this

```
$speakers = $this->model->with('translations')
            ->where('published', true)->withRelatedBrowsers('featuredEvent')->get()
```

Please advise.

Thanks

**Comment by haringsrob** [comment]
Hey @michaeljcoyne you could use with('relatedItems') or make your own getter if you need only a specific one.
